src/views/tutorial-library/components/filters/index.tsx

```diff
@@ -1,20 +1,37 @@
 import { SidebarHorizontalRule } from 'components/sidebar/components'
 
-import { EditionFilter } from '../edition-filter'
-import { ProductFilter } from '../product-filter'
-import { ResourcesFilter } from '../resources-filter'
+import { EditionFilter, EditionFilterProps } from '../edition-filter'
+import { ProductFilter, ProductFilterProps } from '../product-filter'
+import { ResourcesFilter, ResourcesFilterProps } from '../resources-filter'
+import { useFiltersState } from './use-filters-state'
+
+interface TutorialLibraryFiltersProps {
+	products: ProductFilterProps
+	edition: EditionFilterProps
+	resources: ResourcesFilterProps
+}
 
 /**
  * Renders all of the filters for the tutorial library
  */
-export function TutorialLibraryFilters() {
+export function TutorialLibraryFilters({
+	products,
+	edition,
+	resources,
+}: TutorialLibraryFiltersProps) {
 	return (
 		<div>
-			<ProductFilter />
+			<ProductFilter {...products} />
 			<SidebarHorizontalRule />
-			<EditionFilter />
+			<EditionFilter {...edition} />
 			<SidebarHorizontalRule />
-			<ResourcesFilter />
+			<ResourcesFilter {...resources} />
 		</div>
 	)
 }
+
+export function ConnectedTutorialLibraryFilters() {
+	const filtersState = useFiltersState()
+
+	return <TutorialLibraryFilters {...filtersState} />
+}
```

Comment:
I had to decouple the algolia state from the filters components to work around a bug when the hooks get unmounted, resulting in a reset of the search state. No functional change here really.